lib/axon.ex

```diff
@@ -1528,10 +1528,34 @@ defmodule Axon do
   end
 
   @doc """
-  Splits input graph into a container of `n` input graphs
-  along the given axis.
+  Splits input graph into a container of `m` other input graphs 
```

Comment:
```suggestion
  Splits input graph into a container of `m` other graphs 
```

@seanmor5 is this suggestion incorrect?

Reply:
Ohhh yes this one you are correct!